pkg/cmd/server/api/v1/types.go

```diff
@@ -73,6 +73,9 @@ const (
 	ControllersAll = "*"
 )
 
+// FeatureList contains the set of features to enable or disable
```

Comment:
nit: it's just a set of features.  The type doesn't know or care what it's being used for.